clients/mtaclient_v2/mta_v2_client_operations.go

```diff
@@ -0,0 +1,17 @@
+package mtaclient_v2
+
+import (
+	models "github.com/cloudfoundry-incubator/multiapps-cli-plugin/clients/models"
+	strfmt "github.com/go-openapi/strfmt"
+)
+
+// MtaClientOperations drun drun drun x2
```

Comment:
If this is not auto generated comment, it should be deleted

Reply:
Do not delete it - fix it :D